### PR TITLE
Context 1.19

### PR DESCRIPTION
**Pull request by yalosev**
#### Overview

Migration to go-client 0.19.11
Add context to functions

#### What this PR does / why we need it

Shell operator using this versions of go-client

#### Does this PR introduce a user-facing change?

None

```release-note

```

### CODE REVIEW

File: pkg/module_manager/module_manager_test.go [hunk 1]
```diff
@@ -61,7 +62,7 @@ func initModuleManager(t *testing.T, mm *moduleManager, configPath string) {
 		_ = yaml.Unmarshal(cmDataBytes, &cmObj)
 
 		kubeClient := kube.NewFakeKubernetesClient()
-		_, _ = kubeClient.CoreV1().ConfigMaps("default").Create(cmObj)
+		_, _ = kubeClient.CoreV1().ConfigMaps("default").Create(context.TODO(), cmObj, v12.CreateOptions{})
```
Comment (zuzzas) [inline pkg/module_manager/module_manager_test.go:65]: v12 -> metav1

It's a common name for this import in the Kubernetes project.

File: pkg/helm/helm_test.go [hunk 2]
```diff
@@ -141,14 +143,14 @@ func TestHelm(t *testing.T) {
...
-	_, err = kubeClient.CoreV1().Namespaces().Create(testNs)
+	_, err = kubeClient.CoreV1().Namespaces().Create(context.TODO(), testNs, v12.CreateOptions{})
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	sa := &v1.ServiceAccount{}
 	sa.Name = "tiller"
-	_, err = kubeClient.CoreV1().ServiceAccounts(app.Namespace).Create(sa)
+	_, err = kubeClient.CoreV1().ServiceAccounts(app.Namespace).Create(context.TODO(), sa, v12.CreateOptions{})
```
Comment (zuzzas) [inline pkg/helm/helm_test.go:153]: v12 -> metav1

**Review by zuzzas — CHANGES_REQUESTED** [review]

**Review by zuzzas — APPROVED** [review]

File: pkg/kube_config_manager/kube_config_manager.go [hunk 3]
```diff
@@ -148,7 +148,7 @@ func (kcm *kubeConfigManager) changeOrCreateKubeConfig(configChangeFunc func(*v1
 			return err
 		}
 
-		_, err := kcm.KubeClient.CoreV1().ConfigMaps(kcm.Namespace).Update(obj)
+		_, err := kcm.KubeClient.CoreV1().ConfigMaps(kcm.Namespace).Update(context.TODO(), obj, metav1.UpdateOptions{})
```
Comment (nabokihms) [inline pkg/kube_config_manager/kube_config_manager.go:151]: I started thinking that maybe we should add background context with a timeout to all Kubernetes operations. We can leave it as-is for now, though.

**Review by nabokihms — APPROVED** [review]

File: pkg/kube_config_manager/kube_config_manager.go (hunk 3 above)
Comment (zuzzas) [reply pkg/kube_config_manager/kube_config_manager.go:151]: I would advise creating an issue regarding this comment.

**Pull request merged**